.github/workflows/e2e.yml

```diff
@@ -0,0 +1,22 @@
+name: E2E Test
+on: 
+  pull_request:
+jobs:
+  e2e-test:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        go-version: [ '1.22' ]
+    name: E2E Test
+    timeout-minutes: 40
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup Go
+      uses: actions/setup-go@v4.0.0
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: E2E Test
+      shell: bash
+      run: |
+        sudo make e2e
```

Comment:
E2E test would build Kmesh image and push it to the local registry and there is also a `Build Kmesh` step in CI. Can the two be combined into one? Simply delete `Build Kmesh` or let E2E use the result of `Build Kmesh`.

Reply:
The two github workflows are running in different virtual environments. So the image built by the other workflow cannot be used directly.